lib/apivore/rspec_matchers.rb

```diff
@@ -22,8 +22,18 @@ module RspecMatchers
       end
 
       define_method :fetch_master_swagger do
-        req = Net::HTTP.get(master_swagger_host, "/swagger.json")
-        JSON.parse(req)
+        res =
+          if master_swagger_url.starts_with? 'http'
+            Net::HTTP.get_response(URI(master_swagger_url))
+          else
+            Net::HTTP.get_response(master_swagger_url, "/swagger.json")
+          end
+
+        unless res.is_a? Net::HTTPSuccess
+          message = "Master swagger at #{master_swagger_url} not accessible\n"
```

Comment:
Thanks for this! It was frustrating to see unrelated error messages when swagger download was failing.